examples/SDO_to_STEREO_Coordinate_Conversion.py

```diff
@@ -0,0 +1,147 @@
+"""
+===================================
+AIA to STEREO Coordinate Conversion
+===================================
+
```

Comment:
This example requires WCSaxes be installed, right?  If so, I would make that explicit.